data_steward/tools/copy_dataset_to_output_prod.py

```diff
@@ -0,0 +1,204 @@
+"""
+This script copies a dataset from a source project to the output prod project
+and runs the recreate_person tool.
+"""
+
+# Python imports
+import logging
+import argparse
+import re
+
+# Project imports
+from utils import bq, auth, pipeline_logging
+from tools.recreate_person import update_person
+
+SCOPES = [
+    'https://www.googleapis.com/auth/bigquery',
+    'https://www.googleapis.com/auth/devstorage.read_write',
+    'https://www.googleapis.com/auth/cloud-platform'
+]
+LOGGER = logging.getLogger(__name__)
+
+TIER_LIST = ['controlled', 'registered']
+DEID_STAGE_LIST = ['deid', 'base', 'clean']
+
+
+def validate_tier_param(tier):
+    """
+    helper function to validate the tier parameter passed is either 'controlled' or 'registered'
+
+    :param tier: tier parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if tier.lower() not in TIER_LIST:
+        msg = f"Parameter ERROR: {tier} is an incorrect input for the tier parameter, accepted: controlled or " \
+              f"registered"
+        LOGGER.error(msg)
+        raise argparse.ArgumentTypeError(msg)
+
+
+def validate_deid_stage_param(deid_stage):
+    """
+    helper function to validate the deid_stage parameter passed is correct, must be 'deid', 'base' or 'clean'
+
+    :param deid_stage: deid_stage parameter passed through from either a list or command line argument
+    :return: nothing, breaks if not valid
+    """
+    if deid_stage not in DEID_STAGE_LIST:
+        msg = f"Parameter ERROR: {deid_stage} is an incorrect input for the deid_stage parameter, accepted: deid, " \
+              f"base, clean"
+        LOGGER.error(msg)
+        raise argparse.ArgumentTypeError(msg)
```

Comment:
This should already be validated since you're using choices in argparse. A unit test would help validate this.